Patterns/English/English-PhoneNumbers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+NumberReplaceToken: '@builtin.phonenumber'
+ForbiddenPrefixRegex: !simpleRegex
```

Comment:
I think this should be renamed to FalsePositivePrefixRegex. And we can also add a named group to represent what kind of false positive (in this case, card number) as in the future we may have multiple.